packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -216,6 +231,13 @@ const BlockDraggableWrapper = ( { children } ) => {
 
 	return (
 		<>
+			{
+				<DroppingInsertionPoint
+					scroll={ scroll }
+					hasStartedDraggingOver={ hasStartedDraggingOver }
+					targetBlockIndex={ targetBlockIndex }
+				/>
+			}
```

Comment:
Nitpick: We can remove the braces.
```suggestion
			<DroppingInsertionPoint
				scroll={ scroll }
				hasStartedDraggingOver={ hasStartedDraggingOver }
				targetBlockIndex={ targetBlockIndex }
			/>
```

Reply:
Nice catch! I had a check there before and missed removing the braces. Updated in 72db3986f5a17a2665f7141439f92f8804ab407c